pkgs/development/libraries/pstreams/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, fetchgit
+}:
+
+stdenv.mkDerivation rec {
+  pname = "PStreams";
+  version = "d21afef311be496e6495a5e2103eec7a53ecb817";
```

Comment:
This version string is way too long. How about `d21afef` or even `1.0.1-d21afef` ?

Reply:
Even better to stick to the stable release `1.0.1`.
The only commit that comes after doesn't seem vital and may even break programs depending on `pstreams`

Reply:
Good point. The other commits apart from the last one that you've already mentioned are all testing related. Not much point moving beyond 1.0.1.